olive/passes/pytorch/slicegpt.py

```diff
@@ -32,8 +31,8 @@ class SliceGPT(Pass):
     @staticmethod
     def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
         return {
-            "calibration_data_config": PassConfigParam(
-                type_=Union[DataConfig, Dict],
+            "calibration_dataset": PassConfigParam(
+                type_=str,
                 required=True,
                 description=("Data config for Dataset to calibrate and calculate perplexity on."),
```

Comment:
Can you add the supported datasets in the description?

Reply:
Done!